src/main/java/org/jabref/gui/JabRefFrame.java

```diff
@@ -262,7 +266,44 @@ private void initDragAndDrop() {
                             for (Tab libraryTab : tabbedPane.getTabs()) {
                                 if (libraryTab.getId().equals(tab.getId()) &&
                                         !tabbedPane.getSelectionModel().getSelectedItem().equals(libraryTab)) {
-                                    ((LibraryTab) libraryTab).dropEntry(stateManager.getLocalDragboard().getBibEntries());
+                                    LibraryTab libraryTabCasted = (LibraryTab) libraryTab;
+                                    if (dragboard.hasContent(DragAndDropDataFormats.GROUP)) {
+                                        List<String> groupPathToSources = (List<String>) dragboard.getContent(DragAndDropDataFormats.GROUP);
+                                        if (libraryTabCasted.getBibDatabaseContext().getMetaData().getGroups().isEmpty()) {
+                                            // a root (all entries) GroupTreeNode
+                                            GroupTreeNode rootNode = getCurrentLibraryTab().getBibDatabaseContext()
```

Comment:
When I saw this code, I asked root node of what. A better name would be *currentLibraryGroupRoot*.